proto/regen/ecocredit/v1/state.proto

```diff
@@ -108,25 +113,27 @@ message ProjectInfo {
 message BatchInfo {
   option (cosmos.orm.v1alpha1.table) = {
     id : 5,
-    primary_key : {fields : "id", auto_increment : true}
+    primary_key : {fields : "row_id", auto_increment : true}
     index : {id : 1 fields : "batch_denom", unique : true}
     index : {id : 2 fields : "project_id"}
     index : {id : 3 fields : "start_date"}
     index : {id : 4 fields : "issuer"}
   };
 
-  // id is an auto-incrementing integer to succinctly identify the batch
-  uint64 id = 1;
+  // row_id is the numeric table row identifier of the credit batch.
+  uint64 row_id = 1;
```

Comment:
here we don't have "other" `id`, so why renaming? I think ID should be the canonical attribute used when we reference the objects.

Reply:
I think we should be consistent. It might be confusing if some tables use `id` and others use `key` or something else.